source/winUser.py

```diff
@@ -377,10 +377,23 @@ class GUITHREADINFO(Structure):
 # The height of the virtual screen, in pixels.
 SM_CYVIRTUALSCREEN = 79
 
+# Actions associated with ChangeWindowMessageFilterEx
+# https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-changewindowmessagefilterex
+# Adds the message to the filter. This has the effect of allowing the message to be received.
+MSGFLT_ALLOW = 1
+# Removes the message from the filter. This has the effect of blocking the message.
+MSGFLT_DISALLOW = 2
+# Resets the window message filter to the default.
+# Any message allowed globally or process-wide will get through.
+MSGFLT_RESET = 0

```

Comment:
I think over time we should try to move towards enums for these constants. The downside is that the names no longer exactly match the constant names in Windows. 

The upside is that it helps to make it clear what are the accepted values for the argument, it is harder to use a constant intended for a different function. The other benefit is that it becomes easier to trace where in the codebase these values are being used.

```suggestion
class MSGFLT(enum.IntEnum):
	# Actions associated with ChangeWindowMessageFilterEx
	# https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-changewindowmessagefilterex
	# Adds the message to the filter. This has the effect of allowing the message to be received.
	ALLOW = 1
	# Removes the message from the filter. This has the effect of blocking the message.
	DISALLOW = 2
	# Resets the window message filter to the default.
	# Any message allowed globally or process-wide will get through.
	RESET = 0

```

I don't think this is super important, I'll leave this up to you.